src/App/App.js

```diff
@@ -1,38 +1,58 @@
 import React, { Component } from 'react';
 import MovieList from '../MovieList/MovieList';
 import Header from '../Header/Header';
-import movieData from '../mockData';
 import './App.css';
 import MovieDetail from '../MovieDetail/MovieDetail';
+import { getAllMovies, getMovieDetails } from '../api-calls/api-calls';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      selectedMovie: null
+      movies: [],
+      selectedMovie: null,
+      error:''
     }
   }
 
-  handleClick = (id) => {
-    const clickedMovie = this.state.movies.find(movie => movie.id === id);
-    this.setState({ selectedMovie: clickedMovie });
+  componentDidMount() {
+    getAllMovies()
+    .then(data => {
```

Comment:
The syntax difference between setting movies in state and setting an error in state should probably be made consistent. Either use the implicit return for both or neither.